tests/testthat/test_piat_random.R

```diff
@@ -76,18 +76,21 @@ q <- 1 # Number of question
 for (i in sample(0:1, number_items, replace=TRUE)){
   app$expect_ui_text(paste("Question", q, "out of", number_items, "Did the final tone match the note you were imagining? Click here to play Match No match"))
   app$click(i) #0 = No match
-  print(paste0("answer", i))
+  if (interactive()) print(paste0("answer", i))
   q <- q + 1
 }
 
 app$expect_ui_text("You finished the test! Next")
 app$click_next()
 
-# Results
-results <- app$get_results() %>% as.list()
-print(j)
-PIAT_SEM[j] <<- results[["PIAT"]][["ability_sem"]]
-PIAT_ability[j] <<- results[["PIAT"]][["ability"]]
+if (FALSE) {
+  # Results
+  results <- app$get_results() %>% as.list()
+  print(j)
+  PIAT_SEM[j] <<- results[["PIAT"]][["ability_sem"]]
+  PIAT_ability[j] <<- results[["PIAT"]][["ability"]]
```

Comment:
Variable j was actually passed by another function (Random-Tests.R), which called all random tests (CABAT, MPT, MDT and PIAT) in a row. So we don't need it here anymore, if we change file Random-Tests.R (I already did that for CABAT). Then these lines would result:
````
BAT_ability_sem <<- results[["BAT"]][["ability_sem"]]
BAT_ability <<- results[["BAT"]][["ability"]]
````

Reply:
Ah, makes sense! Yes, I didn't really fix this properly haha. Would you like to edit it with your proposed changes? Basically the important thing is that the test runs in isolation (previously it would fail because j was not defined).

Reply:
Sure. Looks like it runs now. Should I correct this in all other test files (tests/testthat/test_mpt_random.R) as well?

Reply:
Great! Yes, that would be good :)

Reply:
Oh, and that last step you need to do is click "Merge pull request" or something like that.

Reply:
> Great! Yes, that would be good :)
Done. :)